eo-runtime/src/main/java/org/eolang/PhDefault.java

```diff
@@ -41,6 +41,10 @@ public class PhDefault implements Phi, Cloneable {
      *  {@link PhDefault#NESTING} to prevent memory leaks. We should either find a place where this
```

Comment:
@karishka1222 it seems that the PR resolves this puzzle, so we should remove it from the source code. Am I missing something?